File: CodeForces/101-200/118/A/118A.go
// Problem 118A: String Task
// https://codeforces.com/contest/118/problem/A
// Executed with: go version go1.19.8 linux/amd64
// Linux version: Linux kernel 6.1.0-10-amd64
// Submitted on: October 23rd, 2025
// Codeforces language used: Go 1.22.2

package main

import (
	"fmt"
	"strings"
)

func main() {
	var w string
	fmt.Scan(&w)
	w = strings.ToLower(w)

	for i := 0; i < len(w); i++ {
		if !(strings.Contains("aeiouy", string(w[i]))) {
			fmt.Print(".")
			fmt.Print(string(w[i]))
		}
	}
	fmt.Print("\n")
}

File: CodeForces/101-200/118/A/118A.php
<?php

// Problem 118A: String Task
// https://codeforces.com/contest/118/problem/A
// Executed with: PHP 8.2.29 (cli) (built: Jul  3 2025 16:16:05) (NTS)
// Linux version: Linux kernel 6.1.0-10-amd64
// Written on: October 24th, 2025
// Codeforces language used: Cannot be submitted

$string = strtolower(trim(fgets(STDIN)));
$vowels = "aeiouy";
$output = "";

for ($x = 0; $x < strlen($string); $x++) {
    if (str_contains($vowels, substr($string, $x, 1))) {
        continue;
    } else {
        $output .= ".";
        $output .= substr($string, $x, 1);
    }
}
echo "$output\n";
?>
